Lib/fractions.py

```diff
@@ -369,7 +371,7 @@ def forward(a, b):
         def reverse(b, a):
             if isinstance(a, numbers.Rational):
                 # Includes ints.
-                return monomorphic_operator(a, b)
+                return fallback_operator(Fraction(a), b)
```

Comment:
Why the fallback_operator here? `Fraction(a)` and `b` are both Fractions, or not?

Reply:
Yes, probably here `monomorphic_operator()` could be restored back.